eviatar/enableFeatures_createUsers.ps1

```diff
@@ -0,0 +1,54 @@
+﻿function Install-WindowsFeatures {
+    param (
+        [Parameter(Mandatory = $true)]
+        [string[]]$Features
+    )
+    
+    foreach ($Feature in $Features) {
+        Write-Host "Installing feature: $Feature"
+        Install-WindowsFeature -Name $Feature -IncludeAllSubFeature -IncludeManagementTools -Verbose

```

Comment:
What happens if this fails
How would you know where to look?

Reply:
If the Install-WindowsFeature command fails, I should implement error handling using a Try-Catch block. This allows you to capture the error and log it or perform additional actions like retrying the installation. For debugging, PowerShell automatically logs errors in the $Error variable, which you can inspect after the script runs.